web/src/pages/admin/createTicket/index.js

```diff
@@ -27,6 +28,38 @@ export default function AdminUserTickets() {
     reloadTickets();
   }, []);
 
+  const setFormsValues = () => {
+    form.setValue('name', user.name);
+    form.setValue('email', user.email);
+    form.setValue('cpf', formatCpf(user.cpf));
+  };
```

Comment:
You could also set the form inputs as `readonly` while the checkbox is checked. Please do not set as `disabled`, since disabled inputs are not sent with the form